charts/feedbacksystem/README.md

```diff
@@ -1,36 +1,120 @@
+| ------------------------------------------- | ----------------------------------------------------------------------------------------------------- | ------- |
+| runner.enabled                              | Should the Container be Enabled                                                                       | true    |
+| runner.config.hmacSecret                    | Secret Used to Generate hmacs for Database Users                                                      | fbs     |
+| runner.config.debug.disableContainerRemoval | Determines whether the spawned Docker container that is used for a check should be kept after its use | false   |
+
+#### Digital Classroom
+
+| Parameter                          | Description                                        | Default                          |
+| ---------------------------------- | -------------------------------------------------- | -------------------------------- |
+| digitalClassroom.enabled           | Should the Container be Enabled                    | true                             |
+| digitalClassroom.config.jwtSecret  | The Used JWT Secret for the Digital Classroom      | 2edb8793d987389e1626918e0ec1dbee |
+| digitalClassroom.config.secret     | Secret used to interact with the Digital Classroom | 1f2a08f5dbd81580a6fb0f645dce3737 |
+| digitalClassroom.config.bbb.url    | Url to the BigBlueButton Sever that should be used | https://bbb.example.org          |
+| digitalClassroom.config.bbb.secret | Secret for the Configured BigBlueButton Server     | 1234                             |
+
+#### Mysql
+
+| Parameter                       | Description                            | Default   |
+| ------------------------------- | -------------------------------------- | --------- |
+| mysql.auth.database             | The name of the Database               | fbs       |
+| mysql.auth.username             | The name of the Database User          | fbs       |
+| mysql.auth.password             | The Password of the Database User      | fbs       |
+| mysql.auth.rootPassword         | The Password of the Database Root User | root      |
+| mysql.primray.existingConfigmap | **TODO**                               | CHANGE-ME |
+
+#### Runner Mysql
+
+| Parameter                             | Description                            | Default   |
+| ------------------------------------- | -------------------------------------- | --------- |
+| runnerMysql.auth.database             | The name of the Database               | fbs       |
+| runnerMysql.auth.username             | The name of the Database User          | fbs       |
+| runnerMysql.auth.password             | The Password of the Database User      | fbs       |
+| runnerMysql.auth.rootPassword         | The Password of the Database Root User | root      |
+| runnerMysql.primray.existingConfigmap | **TODO**                               | CHANGE-ME |
```

Comment:
@Zitrone44 I didn't know exactly what this configuration was for, can you add a description here?